01/5. Switch statement/main.c

```diff
@@ -0,0 +1,20 @@
+#include <stdio.h>
+
+int main(void)
+{
+    int x = 3;
+    switch (x) {
+    case 0:
+        printf("Case 0\n");
+    case 1:

```

Comment:
are we sure we don't want to put any break; here?

some can have breaks while some don't...